src/components/Settings/Settings.react.js

```diff
@@ -685,7 +687,124 @@ class Settings extends Component {
+      avatarRender = (
+        <form
+          style={{ display: 'inline-block', marginTop: '10px' }}
+          onSubmit={e => this.handleAvatarSubmit(e)}
+        >
+          {this.state.isAvatarAdded && (
+            <div>
+              <div className="close-avatar">
+                <Close
+                  style={{ height: '20px', width: '20px', margin: '2px 2px' }}
+                  onClick={this.removeAvatarImage}
+                />
+              </div>
+              <img
+                alt="User Avatar"
+                className="setting-avatar"
+                src={this.state.imagePreviewUrl}
+                onClick={e => this.handleAvatarImageChange(e)}
+              />
+            </div>
+          )}
+          <label htmlFor="file-opener">
+            <div onSubmit={e => this.handleAvatarImageChange(e)}>
+              {!this.state.isAvatarAdded && (
+                <div className="avatar-empty-box">
+                  <div
+                    style={{
+                      margin: '0 auto',
+                      width: '100%',
+                      height: '100%',
+                      display: 'flex',
+                      alignItem: 'centre',
+                    }}
+                  >
+                    <Add
+                      className="avatar-add-button"
+                      style={{
+                        margin: '50px auto',
+                        height: '50px',
+                        width: '50px',
+                      }}
+                    />
+                  </div>
+                </div>
+              )}
+            </div>
+            <input
+              id="file-opener"
+              type="file"
+              className="input-avatar"
+              onChange={e => this.handleAvatarImageChange(e)}
+              accept="image/x-png,image/gif,image/jpeg"
+              style={{ marginTop: '10px' }}
+            />
+          </label>
+          <div
+            style={{ marginTop: '10px', width: '150px' }}
+            className={`upload-btn file-upload-btn${
+              this.state.file && this.state.isAvatarAdded ? '' : '-disabled'
+            }`}
+            title="Upload Avatar"
+          >
+            {this.state.file && this.state.uploadingAvatar ? (
+              <CircularProgress color="#ffffff" size={32} />
+            ) : (
+              <div
+                disabled={!this.state.file}
+                onClick={e => this.handleAvatarSubmit(e)}
+              >
+                Upload Image
+              </div>
+            )}
+          </div>
+        </form>
+      );
+    } else if (this.state.avatarType === 'default') {
+      avatarRender = (
+        <img
+          alt="Default avatar"
+          className="setting-avatar"
+          src={defaultAvatar}
+        />
+      );
+    } else {
+      avatarRender = (
+        <img
+          alt="Gravatar avatar"
+          className="setting-avatar"
+          src={gravatarAvatar}
```

Comment:
It should not be a constant image, it should display the Gravatar image of the user.